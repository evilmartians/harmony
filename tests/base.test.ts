import { expectBuildToMatchSnapshot } from "./utils.ts";
import base from "../scripts/targets/base/base.config.ts";

Deno.test("build base target", async (t) => {
  await expectBuildToMatchSnapshot(t, base);
});

