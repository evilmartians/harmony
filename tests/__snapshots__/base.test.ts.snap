export const snapshot = {};

snapshot[`index.js 1`] = `
'export default {
  red: {
    50: "oklch(0.988281 0.0046875 20)",
    500: "oklch(0.742188 0.151562 20)",
  },
  orange: {
    100: "oklch(0.966797 0.0171875 43.3333)",
  },
};
'
`;

snapshot[`index.cjs 1`] = `
'"use strict";
module.exports = {
  red: {
    50: "oklch(0.988281 0.0046875 20)",
    500: "oklch(0.742188 0.151562 20)",
  },
  orange: {
    100: "oklch(0.966797 0.0171875 43.3333)",
  },
};
Object.defineProperty(exports, "__esModule", { value: true });
'
`;

snapshot[`index.d.ts 1`] = `
"declare namespace _default {
  let red: {
    50: string;
    500: string;
  };
  let orange: {
    100: string;
  };
}
export default _default;
"
`;
